### PR TITLE
Check makefiles env cmd target

### PR DESCRIPTION
**Pull request by chmouel**
* Check that we have all the ./cmd/ binaries in our Makefile variables if we didn't it means upstream added a binary and explain what to do, i.e: 

image

* Remove the `push-images` makefile target since we have mirroring jobs now 💃

### CODE REVIEW

**Comment by chmouel** [comment]
I guess i'll watch the tour de france finish while the `cloud` get fixed : 

image

File: Makefile [hunk 1]
```diff
@@ -44,16 +67,4 @@ generate-release:
 	@./openshift/release/generate-release.sh v$(RELEASE_VERSION)
 .PHONY: generate-release
 
-push-image:
```
Comment (vdemeester) [inline Makefile]: Removing that for ?

Comment (chmouel) [reply Makefile]: image

**Comment by chmouel** [comment]
I pushed to master already, thinking it was merged 🤦‍♂ 

i guess i'll just cancel this one since this will be applied, 

if you still have concerns @vdemeester feel free to ping me directly

**Pull request closed without merging**